gcp-vertex-ai-mojo-scorer/README.md

```diff
@@ -0,0 +1,112 @@
+```shell script
+./gradlew build -PdockerRepositoryPrefix=gcr.io/your/repository
+```
+and the docker image required for GCP Vertex AI will be in the directory `gcp-vertex-ai-mojo-scorer/build`:
+```shell script
+/path/to/dai-deployment-templates/gcp-vertex-ai-mojo-scorer/build/jib-image.tar
+```
+
+* Load the resulting `jib-image.tar` file to docker
+```shell script
+docker load < /path/to/dai-deployment-templates/gcp-vertex-ai-mojo-scorer/build/jib-image.tar
+``` 
+
+* Follow the steps explained here in Google Documentation: https://cloud.google.com/run/docs/building/containers, to 
+push the image to gcr.io.
+
+## Deploying
+
+To deploy the container follow the steps in Google Documentation here to import the model:
+https://cloud.google.com/vertex-ai/docs/general/import-model
+
+There is one requirement for the container. You __MUST__ include the following environment variables:
+* MOJO_GCS_PATH = `gs://path/to/pipeline.mojo`
+* LICENSE_GCS_PATH = `gs://path/to/driverless/ai/license.sig`
+
+If you built the Docker image with the preprocessing script option, you also __MUST__ include the following environment variable:
+* PREPROCESSING_SCRIPT_PATH = `gs://path/to/preprocessing_script.py`
+
+The prediction route will be `/model/score` and the health route will be `/model/id`.
+
+You can then deploy an endpoint once the model has been imported.
+
+## Preprocessing Script
+
+If you are including a Python data preprocessing script, here is an example:
+
+```
+#!/usr/bin/env python
+# coding: utf-8
+import json
+import sys
+
+fileName = str(sys.argv[1])
+
+# Load JSON file with request data
+with open("/tmp/"+fileName) as f:
+    request = json.load(f)
+    
+data = [
+    [69338.0,0.7315255726985329,-0.5847660267950631,0.386082437076677,0.821180632502935,-0.0312602331172659,1.1980865440065802,-0.13618956328652698,0.391451241672127,0.3738906601386479,-0.461823737560995,0.625861374468075,1.07615129929773,0.0718516496509022,0.0620528341180761,0.850326761944984,-1.6339872927690802,1.3467406501738701,-2.7597098786627994,-1.5048495641502198,0.0533811518528488,0.00272572637419962,0.00836864608482666,0.0456492844845917,-0.5780279514412049,0.0478507743996201,0.36036065876766993,0.0224220501004547,0.0282128845008635,157.49],
+    [81081.0,-0.873286664580274,0.5383418573374871,2.65055031553525,0.331972130017534,0.0510277957117218,0.6364468979718391,0.6719139650867261,0.18165642237906401,0.0793536531651418,-0.684037044169426,0.8644669396918221,0.8734041370577021,-0.7131054601776229,-0.494781246049671,-1.94685930894166,-0.6014164255583699,-0.0460304794694537,-0.356068770928204,-0.537032123913058,-0.0619970210088985,-0.00642393408573803,0.32916282786391804,-0.33233023038250503,0.22704204810440198,0.546100808417714,-0.40064788122765005,-0.0896039366748501,-0.166155749695546,38.0]
+]
+
+# Replace data with new samples
+request["instances"] = data
+
+# Dump new request to JSON
+with open("/tmp/"+fileName, 'w') as f:
```

Comment:
So I guess we are sticking with disk io for now. No change necessary. But this maybe needs to be made clear in the docs.